### DISCUSSION TITLE
Problem mit falschen Werten bei der Gesamtleistung die nur durch Neustart zu beheben sind

### DISCUSSION THREAD

**Discussion by B51-solar** [tbnobody/OpenDTU#1195] [Q&A]
Hallo, ich habe eine OpenDTU dort sind 2 Hoymiles Wechselrichter konfiguriert.
1x HM 1500 mit 4x Trina V+425 und direkt Verbund ein HM-300 (zur Nachteinspeisung) an einer 25,6 V LiFePo4 Batterie diese wird von 2x Trina v+425 über einen Vicrton MPPT 100/20 Smart Solar geladen und über den Lastausgang & Straßenlicht Funktion gesteuert. 
Wenn Morgens die Sonne auf geht, unterbricht der Lastausgang des Victron mittels Relais die Stromzufuhr zum HM-300. Die OpenDTU teilt mir mit das der HM-300 nicht mehr antwortet und nur noch der HM-1500 via den Panels einspeist.
Soweit so gut, jedoch wird in der Gesamtleistung des Webinterfaces immer noch die Leistung der Einspeisung angezeigt obwohl sie nicht mehr vorhanden ist. Das der HM-300 nicht mehr einspeist bestätigen auch die Werte die der Tasmota-Lesekopf am Zähler ermittelt. Wenn ich bei der OpenDTU einen  Neustart durchführe ist das Problem bis zum nächsten Sonnenaufgang behoben, dann beginnt das Spiel von vorne. Hat jemand einen Vorschlag zur abhilfe des Problems?

**Comment by jimjackii** [comment]
Das habe ich auch schon beobachtet, daß die "alten" Einspeisewerte noch vorhanden sind und erst mit einem Neustart des WR auf 0 gesetzt werden.
Wahrscheinlich ist das eine Krux. Sofort löschen heißt Wert ist weg, Löschen erst bei Neustart heißt der alte Wert bleibt bis dahin stehen.

Eventuell würde da eine Konfigurationsmöglichkeit helfen? 0 Setzen bei Ein/Aus des WR oder zu bestimmter Zeit.
